pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -559,9 +565,15 @@ def create_server_instance(connection_name):
                 pywbem_server = connections_repo[connection_name]
             except KeyError:
                 raise click.ClickException(
-                    "Connection definition '{}' not found in "
-                    "connections file '{}'".
+                    "Connection name '{}' not found in connections file '{}'".
                     format(connection_name, connections_repo.connections_file))
+            except Exception as exc:
+                click.echo('Fatal error loading connections file: "{0}". '
```

Comment:
1. I suggest simply showing the message of the exception that was caught, and making sure they say everything that is needed to be said.
2. If you want to continue using additional text, then please relax it to comething like "Cannot load connections file".
3. I don't like the broad exception handling. See my comment above on a proper exception hierarchy.

Reply:
**Update:** I'll improve the exception handling of `ConnectionRepository` after this PR is merged, so no need to invest in that as part of this PR. You can ignore my comment above.

Reply:
DONE